### DISCUSSION TITLE
Utility classes

### DISCUSSION THREAD

**Discussion by MMTeech** [picocss/pico#451] [Ideas]
I'm currently using your lightweight CSS framework and I'm thoroughly impressed. I have a suggestion for improvement – could you please consider adding utility classes for margin and padding?

**Comment by vardumper** [comment]
Isn't that something that can easily be achieved by preferably 1) setting the according variables 2) or adding them as you need them.
Pico isn't as lightweight as it is because it brings helper classes that only some people would need.
